README.md

```diff
@@ -5,7 +5,29 @@ Mallet, the minimum wallet, is a command line utility for deploying smart contra
 
 ## Installing
 
-Mallet requires Node.js version 10.4.0 or later. There are more detailed instructions for installing Node.js and Mallet in two places.
+We have tested Mallet with Node.js version `10.16.3`. The repository contains a proper `.nvmrc` file, so you can call [nvm](https://github.com/nvm-sh/nvm) to use this particular version, provided it is already installed in your system.
+
+```
+$ git clone https://github.com/input-output-hk/mallet
+
+$ cd mallet
+
+$ cat .nvmrc
+10.16.3
```

Comment:
Why? It's in the repo? :)

Reply:
It is commited, yes

Reply:
Or did you mean to say this is redundant because it is already committed? I opted for being a bit more verbose to increase awareness about `.nvmrc`.

Reply:
Ok